src/components/CvssSection.vue

```diff
@@ -0,0 +1,88 @@
+<script setup lang="ts">
+import { computed, ref } from 'vue';
+
+import CvssNISTForm from '@/components/CvssNISTForm.vue';
+import LabelDiv from '@/components/widgets/LabelDiv.vue';
+
+import { issuerLabels } from '@/composables/useFlawCvssScores';
+
+import type { ZodFlawCVSSType } from '@/types/zodFlaw';
+
+const props = defineProps<{
+  allCvss: ZodFlawCVSSType[];
+  bugzilla: string;
+  cveid: null | string;
+  cvss: string;
+  highlightedNvdCvss3String: { char: null | string; isHighlighted: boolean }[][];
+  nistcvss: string;
+  shouldDisplayEmailNistForm: boolean;
+  summary: string;
+}>();
+
+const showAllCvss = ref(false);
+
+const otherCvss = computed(() => props.allCvss.filter(cvssItem =>
+  (!(cvssItem.cvss_version === 'V3' && (cvssItem.issuer === 'RH' || cvssItem.issuer === 'NIST')))));
```

Comment:
Since `IssuerEnum` won't change, I wonder if we should use that imported value here `IssuerEnum.NIST`

Reply:
Yes, I guess we could use the native enum in all the issuers.